modules/cnf-cpu-infra-container.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.adoc
+:_content-type: PROCEDURE
+[id="cnf-restricting-cpu-infra-container_{context}"]
+:_context: cnf-master
+= Restricting CPUs for infra and application containers
+
+Generic housekeeping and workload tasks use CPUs in a way that may impact latency-sensitive processes. By default, the container runtime uses all online CPUs to run all containers together, which can result in context switches and spikes in latency. Partitioning the CPUs prevents noisy processes from interfering with latency-sensitive processes by separating them from each other. The following table describes how processes run on a CPU after you have tuned the node using the Performance Addon Operator:
+
+.Process' CPU assignments
```

Comment:
This is too much data to put into the front of a procedure. Please split the module.